Ta/fix wordcloud

anquetos opened the pull request:
# Goal

Remove usage of [word_cloud](https://github.com/amueller/word_cloud) library which was causing a resource warning :

```
ResourceWarning: unclosed file <_io.TextIOWrapper name='C:\\Users\\thoma\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\python-template-Sk64sqGT-py3.12\\Lib\\site-packages\\wordcloud\\stopwords' mode='r' encoding='cp1252'>
  STOPWORDS = set(map(str.strip, open(os.path.join(FILE, 'stopwords')).readlines()))
ResourceWarning: Enable tracemalloc to get the object allocation traceback
```

This issue is still opened on the repository (https://github.com/amueller/word_cloud/issues/764).

# Solution

The word cloud is now created directly with the Pillow library and its `ImageDraw` and `ImageFont` modules.

Since it was necessary to specify the `ttf` file to load in the `ImageDraw.truetype()` function, font management has been changed from Google API to a local ressource folder `app/assets/fonts`.

Other ressources of the project as images or CSS files have also been moved to the `assets` folder.

anquetos committed:
Removed usage of wordcloud libray to generate the wordcloud on home page.

anquetos committed:
Moved "images" and "css" folders into an "assets" folder.

anquetos committed:
Fonts know availabe from the pject "assets" folder instead of Google API.

anquetos committed:
Image size initialized with the function parameters.

anquetos committed:
Updated Poetry dependencies.

gaudinnicolas commented:
@anquetos  , ce serait cool d'intégrer https://github.com/dataforgoodfr/12_taxobservatory_dataviz/pull/45 à ta PR avant merge. d'après ce que j'ai vu il y a aura quelques conflits à gérer mais principalement des paths. et vu le nouveau assets, c'est une bonne occasion. t'en penses quoi?

anquetos commented:
@gaudinnicolas, oui ce serait bien. Par contre je ne sais pas trop comment faire pour l'intégrer, peux-tu m'aiguiller ?

anquetos commented:
@gaudinnicolas, ah ok, tout simplement. Je pensais qu'il y avait moyen de faire un truc via GitHub mais c'est plus simple comme ça.
Je le ferai demain matin.

anquetos committed:
Removed unused images.

anquetos committed:
World map image converted from png to jpg.

anquetos committed:
Company image converted from png to jpg.

anquetos committed:
Removed world map and company png image files.

anquetos committed:
Renamed company image file.

anquetos commented:
# Update

* Added images optimisation (conversion from `png` to `jpg`) proposed on PR #45.
* Removed unused images.
* Renamed illustration image used on "company" page.
